src/main/java/org/jenkinsci/plugins/stashNotifier/StashCredentialMatcher.java

```diff
@@ -4,12 +4,13 @@
 import com.cloudbees.plugins.credentials.CredentialsMatcher;
 import com.cloudbees.plugins.credentials.common.CertificateCredentials;
 import com.cloudbees.plugins.credentials.common.UsernamePasswordCredentials;
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
Is it correct import for `NonNull`? Maybe `import javax.annotation.Nonnull;` ?

Reply:
Better not use `javax.annotation.Nonnull` as there is no official JSR release for it. This is always source of discussions - e.g. see this comment https://github.com/jenkinsci/display-url-api-plugin/pull/19#discussion_r233771043. As written in the commit message I am using exactly the same annotation as the implemented interface is using.